src/assets/icons/ThumbFilled.js

```diff
@@ -0,0 +1,14 @@
+export const ThumbFilledIcon = (props) => (
```

Comment:
Did you optimize SVG files? 
[SVG optimizer](https://jakearchibald.github.io/svgomg/)

Reply:
Optimized SVGs commited